.github/workflows/merge-commits-to-next-branch.yml

```diff
@@ -0,0 +1,134 @@
+name: Merge commits to next branch
+
+on:
+  schedule:
+    - cron: "0 */2 * * *"
+
+env:
+  COMMIT_INFO: commit-info
+
+jobs:
+  dequeue:
+    runs-on: ubuntu-latest
+    steps:
+      steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm ci
+      - run: npm install -g @azure/storage-queue
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const { QueueClient, QueueServiceClient } = require('@azure/storage-queue')
+            const connectionString = `${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}`;
+            const queueName = `${{ secrets.AZURE_QUEUE_NAME }}`;
+            const queueServiceClient = QueueServiceClient.fromConnectionString(connectionString);
+            const queueClient = queueServiceClient.getQueueClient(queueName);
+            var receivedMessages = await queueClient.receiveMessages();
+            const message = receivedMessages.receivedMessageItems[0];
+            $COMMIT_INFO = JSON.parse(message.messageText);
+  create-branch:
+    runs-on: ubuntu-latest
+    if: ( $COMMIT_INFO.label == 'queued' )
+    steps:
+      - name: Create branch
+        uses: actions/github-script@7a5c598405937d486b0331594b5da2b14db670da # pin@v6
+        with:
+          script: |
+            git fetch origin
+            git checkout test_next
```

Comment:
Consider making the target branch a variable that can be configured in the workflow.